src/applications/vaos/appointment-list/sass/styles.scss

```diff
@@ -82,6 +82,13 @@
   }
 }
 
+.vaos-appts__breadcrumbs{
+  padding: 2em 0;
+}
+.vaos-appts__breadcrumbs li {
+padding: 0em !important;
```

Comment:
Missing some indentation here

Reply:
Strange there is no linting warning over indentation

Reply:
Yeah, I don't think the css linting is set up all that well in vets-website.